master/internal/resourcemanagers/priority.go

```diff
@@ -392,18 +333,13 @@ func comparePositions(a, b *sproto.AllocateRequest, groups map[*actor.Ref]*group
 func sortTasksWithPosition(
```

Comment:
non-blocking: we can do this in a later commit, but it might be good to combine this fn with sortTasksWithPositionAndTimestamp